src/server/src/Server.cpp

```diff
@@ -160,7 +174,6 @@ std::string Server::set_force(const double f_x, const double f_y, const double f
     return "OK";
 }
 
-
```

Comment:
The adopted convention is having two spaces between each function, can you please put it back? :)

Reply:
done as well